model/map/src/main/java/org/keycloak/models/map/authorization/MapAuthorizationStoreFactory.java

```diff
@@ -54,7 +55,7 @@
     @Override
     public StoreFactory create(KeycloakSession session) {
         MapStorageProviderFactory storageProviderFactory = (MapStorageProviderFactory) getComponentFactory(session.getKeycloakSessionFactory(),
-          MapStorageProvider.class, storageConfigScope, MapStorageSpi.NAME);
+          MapStorageProvider.class, storageConfigScope, MapStorageSpi.NAME, "authorizationPersister");
```

Comment:
Where does this constant originate from?

Reply:
Right now it is coming from the [SPI](https://github.com/keycloak/keycloak/blob/main/server-spi-private%2Fsrc%2Fmain%2Fjava%2Forg%2Fkeycloak%2Fauthorization%2Fstore%2FStoreFactorySpi.java#L31) name. For authorization the different areas are not currently separated, they are all initialized to use the same storage factory. I don't know if we will change that in the future so that the different authz models will be configurable independently with respect to the storage factory, but for now they all share the same so I decided to use the SPI name.

Reply:
If we won't be splitting the different authz models, then perhaps a better name for the authz storage would be simply `authorization`.